model/src/main/java/org/cbioportal/genome_nexus/model/VariantAnnotation.java

```diff
@@ -61,6 +61,7 @@
     private Integer strand;
     private String mostSevereConsequence;
     private List<ColocatedVariant> colocatedVariants;
+    private List<IntergenicConsequences> intergenic_consequences;
```

Comment:
@leexgh i guess this works now to make the integration test pass, but it is not consistent with the other member names. Would be better to change the integration test instead of changing the model to fit the test

Reply:
@inodb yes it is, I already change the test and model name to keep consistent. Thanks!